### ISSUE TITLE
Warning and Print info distorted on Retina windows displays

### ISSUE THREAD

**Issue by pacaj2am** [prusa3d/PrusaControl#46]
Warning and Print info distorted on Retina windows displays.

Probably other items on screen too small (button save gcode, back, buttons to arrange, rotate - 2nd screenshot).

See attached images. Done on Microsoft Surface Pro4 (Screen: 12.3” PixelSense Display , Resolution: 2736 x 1824 (267 PPI))
![prusa-control](https://user-images.githubusercontent.com/1917365/30685697-0ea9a8fc-9eb6-11e7-9ee8-5ed98d13817f.png)
![prusa-control2](https://user-images.githubusercontent.com/1917365/30685823-78b0c7e4-9eb6-11e7-9535-091f3b526fa8.png)

**Comment by jarnold** [comment]
![prusacontrol-printinfo](https://user-images.githubusercontent.com/18742/36639193-26782cde-19bc-11e8-970e-f4caa8cabfe1.PNG)

Similar issue on Windows 10 running at 3840 x 2160 running at any scaling percentage.

**Comment by prusa3d-bb** [comment]
Hello,
Thank you for your feedback. We will take care of this issue as soon as possible.
Kind regards


—
Lukáš Vacek
Customer Support


PRUSA Research
+420 222 263 718
+421 220 570 305
188/7a Partyzánská, 17000, Prague
shop.prusa3D.com


For simple troubleshooting, please use: http://help.prusa3d.com


Please rate our reply ...be generous :-)

--- original message ---
On Sun, Feb 25, 2018 at 08:41 am, <notifications@github.com> jarnold wrote:



Similar issue on Windows 10 running at 3840 x 2160 running at any scaling percentage.

—

You are receiving this because you are subscribed to this thread.

Reply to this email directly, view it on GitHub, or mute the thread.
--- end of original message ---

**Comment by iandouglas** [comment]
Yes, it would be nice to get this fixed as well, I can't easily change rotations or placement values on a Retina display:

<img width="185" alt="screen shot 2018-03-18 at 12 38 49 pm" src="https://user-images.githubusercontent.com/168030/37569546-694b3eb8-2aa9-11e8-846b-85a81be5fcc1.png">

**Comment by prusa3d-bb** [comment]
Thank you very much, I will forward this info further also.
Kind regards


—
Lukáš Vacek
Customer Support


PRUSA Research
+420 222 263 718
+421 220 570 305
188/7a Partyzánská, 17000, Prague
shop.prusa3D.com


For simple troubleshooting, please use: http://help.prusa3d.com


Please rate our reply ...be generous :-)

--- original message ---
On Sun, Mar 18, 2018 at 07:39 pm, <notifications@github.com> ian douglas wrote:


Yes, it would be nice to get this fixed as well, I can't easily change rotations or placement values on a Retina display:


—

You are receiving this because you commented.

Reply to this email directly, view it on GitHub, or mute the thread.
--- end of original message ---

**Comment by WoodWingStudio** [comment]
I'm having the same issue.  Windows 10, Microsoft Surface Book, 3000x2000 resolution


![prusacontrol](https://user-images.githubusercontent.com/38448855/38852355-65a8d7de-41ce-11e8-90b8-3d88a2d6c93d.jpg)
